index.html

```diff
@@ -6254,6 +6257,32 @@ <h2>Acknowledgements</h2>
 Christopher Allen, Heather Vescent, and Wayne Chang.
     </p>
 
+    <p>
+The Working Group thanks the following individuals for their contributions to
+this specification (in alphabetical order): Denis Ah-Kang, Nacho Alamillo,
+Christopher Allen, Joe Andrieu, Antonio, Phil Archer, George Aristy, Baha, Juan
+Benet, BigBlueHat, Dan Bolser, Chris Boscolo, Pelle Braendgaard, Daniel Buchner,
+Daniel Burnett, By_caballero, Tim Cappalli, Melvin Carvalho, David Chadwick,
```

Comment:
Oops I think there's a people.json somewhere I need to update.  Can I get by_caballero changed to Juan Caballero or do I have to open a separate PR?